src/member/templatetags/member_extras.py

```diff
@@ -18,10 +19,17 @@ def dish_size(context, day):
 @register.filter
 def readable_prefs(value):
     if value is None:
-        return ''
+        return _("Not properly configured.")
 
     str = ''
     for component, label in COMPONENT_GROUP_CHOICES:
-        str += '{} {}, '.format(value.get(component, 0), label)
+        count = value.get(component)
+        if count:  # more than 0
+            str += '{} {}, '.format(count, label)
 
```

Comment:
We should display the size as well. For instance: 1 Large Main Dish, 2 Fruit Salad. The size is only applicable for the main dish.